RadixWallet/Core/DesignSystem/Components/AppTextField.swift

```diff
@@ -44,6 +44,8 @@ public struct AppTextField<FocusValue: Hashable, Accessory: View, InnerAccessory
 	let accessory: Accessory
 	let innerAccesory: InnerAccessory
 
+	@State private var isFocused: Bool = false
```

Comment:
My first approach was to use `@Environment(\.isFocused)`, but it didn't work. Therefore, I went with this custom solution that we manually updated the flag with the `onEditingChanged` callback of the `TextField`. 

I think we could be smarter and use our `Focus` and some callback when it changes, but given such property is optional, it would be tricky handling it when `focus = nil`. Open to suggestions if someone comes up with a better idea for this.